test/api/support/session.js

```diff
@@ -30,6 +59,9 @@ const session = (user) => {
         .replace(/=+$/, '')}`;
       return `${config.session.key}=s%3A${signedSessionKey}`;
     });
-};
+}
 
-module.exports = session;
+module.exports = {
+  authenticatedSession,
+  unauthenticatedSession,
+};
```

Comment:
modified this file to export two different session functions for testing: an authenticated one, and an unauthenticated one. The unauthenticated one is so that we can get through the csrf protection middleware in order to actually test that authenticate is properly working.